application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -34,6 +36,11 @@
 public class TeXCommand extends SlashCommandAdapter {
 
     private static final String LATEX_OPTION = "latex";
+    // everything between 2 $'s
+    private static final String MATH_REGION = "(\\$\\S+\\$)";
```

Comment:
`\\S+` is wrong. That way it wouldnt support whitespaces but the tex could as well contain whitespaces. You should accept anything in here. So `.+`.